app/src/main/java/com/example/vshcheglov/webshop/presentation/main/MainActivity.kt

```diff
@@ -179,9 +203,23 @@ class MainActivity : NucleusAppCompatActivity<MainPresenter>(), MainPresenter.Ma
         startActivityForResult(openInChooser, PICK_IMAGE_REQUEST_CODE)
     }
 
-    override fun setUserAvatarImage(bitmap: Bitmap) {
+    private fun setUserAvatarImage(bitmap: Bitmap?) {
         navHeaderUserImage = navMainHeader.findViewById(R.id.navHeaderUserImage)
-        Glide.with(this).load(bitmap).apply(RequestOptions.circleCropTransform()).into(navHeaderUserImage)
+        if (bitmap == null) {
+            Glide.with(this).load(R.drawable.profile_avatar_placeholder_large)
```

Comment:
For image loader libs create separate class like so
`
object ImageLoaderManager {

    fun loadImage(imageView: ImageView, url: String?, @DrawableRes placeholderId: Int) {
        Glide.with(imageView).load(url)
            .apply(setupRequestOptions(placeholderId))
            .into(imageView)
    }

    fun loadCircularImage(imageView: ImageView, url: String?, @DrawableRes placeholderId: Int) {
        Glide.with(imageView).load(url)
            .apply(setupRequestOptions(placeholderId).circleCrop())
            .into(imageView)
    }

    private fun setupRequestOptions(placeholderId: Int): RequestOptions {
        return RequestOptions()
            .diskCacheStrategy(DiskCacheStrategy.RESOURCE)
            .onlyRetrieveFromCache(false)
            .placeholder(placeholderId)
            .dontAnimate()
            .downsample(DownsampleStrategy.CENTER_OUTSIDE)
            .centerCrop()
    }
}

`